unittests/AST/ArithmeticEvaluator.cpp

```diff
@@ -335,36 +340,8 @@ TEST(ArithmeticEvaluator, Cycle) {
                                      arithmeticRequestFunctions);
 
   // Evaluate when there is a cycle.
-  UncachedEvaluationRule::brokeCycle = false;
```

Comment:
I didn't quite know what to do with these tests. The broken cycle value won't be cached anymore in the internally cached rule, and the tests for cycle breaking don't apply anymore.

Reply:
It's better not to cache the broken cycle value; I'd say the tests are wrong and it's fine to change them.